playbooks/roles/beaver/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# the pip packages to install
+beaver_pip_packages:
+  - "beaver==33.3.0"
+
+# the beaver daemon settings
+beaver_daemon_log: "/var/log/beaver/beaver.log"
+beaver_daemon_pid: "/var/run/beaver/beaver.pid"
```

Comment:
Did you fix this?  I've had problems getting beaver running due to permissions issues creating the pid file in that directory.